From e13108d4be405bfd6fba91ddd0832e446a8e6dde Mon Sep 17 00:00:00 2001
Message-Id: <e13108d4be405bfd6fba91ddd0832e446a8e6dde.1228954349.git.christian@jaeger.mine.nu>
In-Reply-To: <030d49bbacadc03ef604478be7d0582bd2703cf7.1228954349.git.christian@jaeger.mine.nu>
References: <030d49bbacadc03ef604478be7d0582bd2703cf7.1228954349.git.christian@jaeger.mine.nu>
From: Christian Jaeger <christian@jaeger.mine.nu>
Date: Wed, 10 Dec 2008 23:15:38 +0100
Subject: [PATCH] Fix?: mkdir -p "$DESTDIR/conf/conf.d"

Note that this seems only be needed because I'm testing with the
cryptroot script alone, possibly in real usage this directory already
exists?

Signed-off-by: Christian Jaeger <christian@jaeger.mine.nu>
---
 debian/initramfs/cryptroot-hook |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/debian/initramfs/cryptroot-hook b/debian/initramfs/cryptroot-hook
index 9e25d09..096c5bb 100644
--- a/debian/initramfs/cryptroot-hook
+++ b/debian/initramfs/cryptroot-hook
@@ -351,6 +351,7 @@ add_device() {
 		if ! get_device_opts "$node" "$opts"; then
 			continue
 		fi
+		mkdir -p "$DESTDIR/conf/conf.d"
 		echo "$OPTIONS" >> "$DESTDIR/conf/conf.d/cryptroot"
 
 		# If we have a keyscript, make sure it is included
-- 
1.6.0.5

